package.json

```diff
@@ -32,18 +32,12 @@
     "postinstall": "ngcc --properties es2015 browser module main --first-only --create-ivy-entry-points && node ./decorate-angular-cli.js && husky install",
     "workspace-generator": "nx workspace-generator"
   },
-  "husky": {
-    "hooks": {
-      "pre-commit": "lint-staged",
-      "commit-msg": "commitlint -e HUSKY_GIT_PARAMS"
-    }
-  },
   "lint-staged": {
     "libs/**/!(images)/**/*.{ts,html,scss,md,json}": [
       "yarn affected:lint -- --uncommitted --fix --parallel"
     ],
     "libs/**/*.{ts,html,scss,md,json}": [
-      "prettier --write"
+      "nx format:write --uncommitted"
     ]
```

Comment:
I think another problem is that we limit this to files inside `libs/` I think we need to get rid of the path constraint entirely and let everything being linted/formatted

Reply:
that's done as well 🫡